migrations/2018-04-05-192744_create_refresh_todays_recent_crate_downloads/up.sql

```diff
@@ -0,0 +1,9 @@
+CREATE FUNCTION refresh_todays_recent_crate_downloads() RETURNS void AS $$
+BEGIN
+  DROP INDEX IF EXISTS todays_recent_crate_downloads;
+  EXECUTE 'CREATE INDEX todays_recent_crate_downloads
+   ON crate_downloads (date)
+   WHERE date > date(''' || CURRENT_DATE::text || '''::date - INTERVAL ''90 days'')';
```

Comment:
Does the WHERE clause in the query need to match to ensure the index is used?  Especially regarding `CURRENT_TIMESTAMP` vs `CURRENT_DATE`.  The query is currently:

`WHERE crate_downloads.date > date(CURRENT_TIMESTAMP - INTERVAL '90 days')`

Reply:
I confirmed that the query as it is written today will use this index before opening the PR